### PR TITLE
Reworks Currencies (Galactic Credit, Coalition Dinar & Company Scrip)

### PR DESCRIPTION
**Pull request by Stutternov**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Adds Galactic Credits, Coalition Dinar & renames base credits to Scrip; implying it's Liberty Group company scrip. Also adds in a vendor for a dungeon that takes Coalition Dinar and dispenses mid-tier to high-tier gear from it. 

Sprites used for Galactic Credits are from Mojave Sun's currency.dmi with some edits to them. I believe made by Inferred Baron. If anyone knows for certain who sprited these besides Inferred, please correct me. Thank you.

image
image
image

<hr>
</details>

## Changelog
:cl:
tweaks: Renames credits to scrip; implying Liberty Group subsidiaries use company scrip.
adds: Galactic Credits, a 10 -> 1 conversion currency to Scrip (regular credits). Now used by Simulacrum vendors.
adds: Coalition Dinar, a 'worthless' currency as inflated as scrip. Used in special Car-Van Coalition vendors.
adds: Car-Van Coalition vendor to be put in dungeon. Players rewarded for killing merc-mobs with Coalition Dinar to use in this machine.
/:cl:

### CODE REVIEW

File: code/modules/economy/cash.dm [hunk 1]
```diff
@@ -1,8 +1,8 @@
 //#define CASH_PER_STAT 1000 // The cost of a single level of a statistic
 
 /obj/item/spacecash
-	name = "0 credit"
-	desc = "It's worth 0 credits."
+	name = "0 scrip"
+	desc = "It's worth 0 scrip."
```
Comment (ThePainkiller) [inline code/modules/economy/cash.dm:5]:
```suggestion
	desc = "It's worth 0 scrips."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/game/machinery/vendor/coalition_vending.dm [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/obj/machinery/vending/carvan
+	name = "Car-Van Coalition Vendor"
+	desc = "An armory vendor sporting Car-Van and Coalition symbols across it. It appears to be rusted with time; probably an old model for a dropship of some sort."
+	icon = 'icons/obj/machines/one_star/vending.dmi'
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/coalition_vending.dm:4]:
```suggestion
```

File: code/game/machinery/vendor/coalition_vending.dm [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	icon_state = "vendor_guns"
+	icon_vend = "vendor_printing"
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/coalition_vending.dm:6]: their is not carvan_printing

Comment (Stutternov) [reply code/game/machinery/vendor/coalition_vending.dm]: Fixed

**Pull request merged**